src/java/detectors/cipher_update_aad/CipherUpdateAADCompliant.java

```diff
@@ -0,0 +1,24 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=java-cipherupdateaad@v1.0 defects=0}
+import javax.crypto.SecretKey;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.GeneralSecurityException;
+
+public class CipherUpdateAADCompliant {
+
+    // Compliant: Cipher initialized with Additional Authenticated Data.
```

Comment:
Please maintain consistent casing. Should be `... with additional authenticated data.`